dbt/models/default/default.vw_pin_universe.sql

```diff
@@ -50,7 +50,7 @@ SELECT
     sp.y_3435,
 
     -- Corner lot indicator
-    lot.is_corner_lot AS ccao_is_corner_lot,
+    COALESCE(lot.is_corner_lot, FALSE) AS ccao_is_corner_lot,
```

Comment:
The `COALESCE` here is needed because the `spatial.corner` table now _only_ contains corners, i.e. it does not contain all PIN10 records. So, anything not in `spatial.corner` is assumed to be not-a-corner.